app/enrichment/app.py

```diff
@@ -262,9 +258,7 @@ def get_tags_and_upload_to_cosmos(blob_service_client, blob_path):
     """ Gets the tags from the blob metadata and uploads them to cosmos db"""
     file_name, file_extension, file_directory = utilities_helper.get_filename_and_extension(blob_path)
     path = file_directory + file_name + file_extension
-    blob_client = blob_service_client.get_blob_client(
-        container=ENV["AZURE_BLOB_STORAGE_UPLOAD_CONTAINER"],
-        blob=path)
+    blob_client = blob_service_client.get_blob_client(blob=path)
```

Comment:
is the removal of `container=ENV["AZURE_BLOB_STORAGE_UPLOAD_CONTAINER"],` correct here? Shouldn't this container property still be here?

Reply:
from my friendly neighborhood copilot...

In the highlighted code, you are using the get_blob_client method of the BlobServiceClient class to get a BlobClient for a specific blob. This method requires the blob path, which includes the container name and the blob name.

If you already have the BlobServiceClient instance and the full blob path (including the container name), you don't need to create a ContainerClient. The BlobServiceClient.get_blob_client method will handle this for you.

However, if you only have the blob name and the ContainerClient instance (which is specific to a container), you can use the ContainerClient.get_blob_client method to get a BlobClient for a specific blob.

So, in your case, you don't need to specify a ContainerClient as long as your blob path includes the container name.